Latest serde breaks cloudevents-sdk

Updating serde to latest breaks cloudevents-sdk

```bash
cargo build
   Compiling proc-macro2 v1.0.24
   Compiling unicode-xid v0.2.1
   Compiling syn v1.0.58
   Compiling autocfg v1.0.1
   Compiling libc v0.2.82
   Compiling serde_derive v1.0.119
   Compiling serde v1.0.119
   Compiling tinyvec_macros v0.1.0
   Compiling matches v0.1.8
   Compiling doc-comment v0.3.3
   Compiling getrandom v0.2.1
   Compiling ryu v1.0.5
   Compiling serde_json v1.0.61
   Compiling percent-encoding v2.1.0
   Compiling cfg-if v1.0.0
   Compiling match_cfg v0.1.0
   Compiling itoa v0.4.7
   Compiling base64 v0.12.3
   Compiling unicode-bidi v0.3.4
   Compiling tinyvec v1.1.0
   Compiling form_urlencoded v1.0.0
   Compiling num-traits v0.2.14
   Compiling num-integer v0.1.44
   Compiling quote v1.0.8
   Compiling time v0.1.43
   Compiling hostname v0.3.1
   Compiling uuid v0.8.2
   Compiling unicode-normalization v0.1.16
   Compiling ordered-float v1.1.1
   Compiling idna v0.2.0
   Compiling snafu-derive v0.6.10
   Compiling delegate-attr v0.2.9
   Compiling snafu v0.6.10
   Compiling url v2.2.0
   Compiling chrono v0.4.19
   Compiling serde-value v0.6.0
   Compiling cloudevents-sdk v0.3.0
error[E0603]: module `export` is private
   --> /home/awebber/.cargo/registry/src/github.com-1ecc6299db9ec823/cloudevents-sdk-0.3.0/src/event/data.rs:1:12
    |
1   | use serde::export::Formatter;
    |            ^^^^^^ private module
    |
note: the module `export` is defined here
   --> /home/awebber/.cargo/registry/src/github.com-1ecc6299db9ec823/serde-1.0.119/src/lib.rs:275:5
    |
275 | use self::__private as export;
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0603]: module `export` is private
   --> /home/awebber/.cargo/registry/src/github.com-1ecc6299db9ec823/cloudevents-sdk-0.3.0/src/event/spec_version.rs:2:12
    |
2   | use serde::export::Formatter;
    |            ^^^^^^ private module
    |
note: the module `export` is defined here
   --> /home/awebber/.cargo/registry/src/github.com-1ecc6299db9ec823/serde-1.0.119/src/lib.rs:275:5
    |
275 | use self::__private as export;
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0603`.
error: could not compile `cloudevents-sdk`

To learn more, run the command again with --verbose.
```

Current workaround is rolling back to serde version 1.0.118

See related fix https://github.com/teloxide/teloxide/issues/328

```bash
[dependencies]
cloudevents-sdk = "0.3.0"
serde = { version="=1.0.118", features = ["derive"] }
```

--- a/src/event/data.rs
+++ b/src/event/data.rs
@@ -1,7 +1,7 @@
-use serde::export::Formatter;
 use serde_json::Value;
 use std::convert::TryFrom;
 use std::fmt;
+use std::fmt::Formatter;
 
 /// Event [data attribute](https://github.com/cloudevents/spec/blob/master/spec.md#event-data) representation
 #[derive(PartialEq, Eq, Debug, Clone)]

--- a/src/event/spec_version.rs
+++ b/src/event/spec_version.rs
@@ -1,7 +1,7 @@
 use super::{v03, v10};
-use serde::export::Formatter;
 use std::convert::TryFrom;
 use std::fmt;
+use std::fmt::Formatter;
 
 pub(crate) const SPEC_VERSIONS: [&str; 2] = ["0.3", "1.0"];
 

